src/lib/handleAuth/handleAuth.ts

```diff
@@ -35,3 +41,17 @@ export async function handleAuth({
 	}
 	throw redirect(302, url.toString());
 }
+
+const storePostRedirectUrl = (options: Record<string, string | number>) => {
```

Comment:
@prokawsar could we call this `storePostLoginRedirectUrl` to be consistent with the NextJS SDK - also enables us to do `post-logout-redirect-url` later if we want to